### PR TITLE
Added weather details info to the screen

### PR DESCRIPTION
**Pull request by Vagik**

### CODE REVIEW

File: data/src/main/java/com/ait/data/openweather/utilities/OwMapper.kt [hunk 1]
```diff
@@ -32,14 +35,29 @@ internal class OwMapper {
         )
     }
 
-    fun convertToDayForecastedWeather(currentWeatherInfo: OwCurrentWeatherInfo): CurrentWeather {
+    fun convertToHourForecastedWeather(hourWeatherInfo: OwHourWeatherInfo): HourForecastedWeather {
+        return HourForecastedWeather(
+            date = Date(hourWeatherInfo.forecastedDateTime * 1000),
```
Comment (ConstAlpha) [inline data/src/main/java/com/ait/data/openweather/utilities/OwMapper.kt:40]: Can be moved to extension function, like ```hourWeatherInfo.forecastedDateTime.toDate()```

File: ui/standard-ui/src/main/java/com/ait/ui/standard/viewmodel/StandardUiViewModel.kt [hunk 2]
```diff
@@ -22,31 +23,45 @@ class StandardUiViewModel(
...
+            else -> TimeOfDay.NIGHT
+        }
+        if (this.dayTime.value != timeOfDay) {
+            this.dayTime.value = timeOfDay
+        }
     }
 
     private suspend fun loadForecast() {
-        // New York
+        // Nizhny Novgorod
```
Comment (ConstAlpha) [inline ui/standard-ui/src/main/java/com/ait/ui/standard/viewmodel/StandardUiViewModel.kt:53]: Sortirovka?

File: ui/standard-ui/src/main/java/com/ait/ui/standard/view/StandardUiFragment.kt [hunk 3]
```diff
@@ -17,12 +22,19 @@ class StandardUiFragment : Fragment() {
...
-    ): View = FragmentStandardUiBinding.inflate(
-        inflater,
-        container,
-        false
-    ).also {
-        it.lifecycleOwner = viewLifecycleOwner
-        it.viewModel = viewModel
-    }.root
+    ): View  {
+        return FragmentStandardUiBinding.inflate(
```
Comment (ConstAlpha) [inline ui/standard-ui/src/main/java/com/ait/ui/standard/view/StandardUiFragment.kt]: Why did you move assignment into inner block?
Code without ```return``` has one line less and the same readability

File: ui/standard-ui/src/main/java/com/ait/ui/standard/view/HourlyWeatherItemView.kt [hunk 4]
```diff
@@ -0,0 +1,31 @@
+package com.ait.ui.standard.view
+
+import android.content.Context
+import android.view.LayoutInflater
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.constraintlayout.widget.ConstraintLayout
+import com.ait.ui.standard.R
+
+class HourlyWeatherItemView(
```
Comment (ConstAlpha) [inline ui/standard-ui/src/main/java/com/ait/ui/standard/view/HourlyWeatherItemView.kt]: Seems like we don't need custom views
Just use inflating in view holders
It's ok if they wont be assigned to ViewModel. We'll do it later

**Review by ConstAlpha — CHANGES_REQUESTED** [review]

File: data/src/main/java/com/ait/data/openweather/utilities/OwMapper.kt (hunk 1 above)
Comment (Vagik) [reply data/src/main/java/com/ait/data/openweather/utilities/OwMapper.kt:40]: Extension function for which class? forecastedDateTime has "Long" type

File: ui/standard-ui/src/main/java/com/ait/ui/standard/view/StandardUiFragment.kt (hunk 3 above)
Comment (Vagik) [reply ui/standard-ui/src/main/java/com/ait/ui/standard/view/StandardUiFragment.kt]: Reverted

File: ui/standard-ui/src/main/java/com/ait/ui/standard/view/HourlyWeatherItemView.kt (hunk 4 above)
Comment (Vagik) [reply ui/standard-ui/src/main/java/com/ait/ui/standard/view/HourlyWeatherItemView.kt]: Removed views classes and moved the code to view holders

File: ui/standard-ui/src/main/java/com/ait/ui/standard/viewmodel/StandardUiViewModel.kt (hunk 2 above)
Comment (Vagik) [reply ui/standard-ui/src/main/java/com/ait/ui/standard/viewmodel/StandardUiViewModel.kt:53]: Ne xochu

File: data/src/main/java/com/ait/data/openweather/model/OwCommonWeatherInfo.kt [hunk 5]
```diff
@@ -2,11 +2,12 @@ package com.ait.data.openweather.model
...
-data class OwCommonWeatherInfo (
+data class OwCommonWeatherInfo(
     @SerializedName("lat") val latitude: Double?,
     @SerializedName("lon") val longitude: Double?,
     @SerializedName("timezone") val timeZone: String,
     @SerializedName("timezone_offset") val timeZoneOffset: Int,
     @SerializedName("current") val current: OwCurrentWeatherInfo,
-    @SerializedName("daily") val daily: List<OwDayWeatherInfo>
+    @SerializedName("daily") val daily: List<OwDayWeatherInfo>,
+    @SerializedName("hourly") val hourly: List<OwHourWeatherInfo>
```
Comment (ConstAlpha) [inline data/src/main/java/com/ait/data/openweather/model/OwCommonWeatherInfo.kt:12]: It's good practice to leave а comma as last symbol in data classes/named arguments to avoid new changes if arguments will be changed

File: data/src/main/java/com/ait/data/openweather/model/OwCurrentWeatherInfo.kt [hunk 6]
```diff
@@ -11,5 +11,6 @@ data class OwCurrentWeatherInfo (
     @SerializedName("humidity") val humidity: Int,
     @SerializedName("wind_speed") val windSpeed: Double,
     @SerializedName("weather") val weather: List<OwWeather>,
-    @SerializedName("clouds") val clouds: Int
+    @SerializedName("clouds") val clouds: Int,
+    @SerializedName("feels_like") val feelsLike: Double,
```
Comment (ConstAlpha) [inline data/src/main/java/com/ait/data/openweather/model/OwCurrentWeatherInfo.kt:15]: Like here

File: data/src/main/java/com/ait/data/openweather/utilities/OwMapper.kt (hunk 1 above)
Comment (ConstAlpha) [reply data/src/main/java/com/ait/data/openweather/utilities/OwMapper.kt:40]: For Long :)
```fun Long.toDate()```

File: ui/standard-ui/build.gradle [hunk 7]
```diff
@@ -40,6 +40,8 @@ dependencies {
     implementation "androidx.appcompat:appcompat:$appcompat_version"
     implementation "androidx.fragment:fragment-ktx:$fragment_ktx_version"
     implementation "com.google.android.material:material:$material_version"
+    implementation "androidx.coordinatorlayout:coordinatorlayout:1.1.0"
+    implementation "androidx.compose.material:material:1.0.0-beta08"
```
Comment (ConstAlpha) [inline ui/standard-ui/build.gradle:44]: Suddenly

File: ui/standard-ui/src/main/java/com/ait/ui/standard/view/StandardUiFragment.kt [hunk 8]
```diff
@@ -17,11 +22,17 @@ class StandardUiFragment : Fragment() {
...
         container: ViewGroup?,
         savedInstanceState: Bundle?
-    ): View = FragmentStandardUiBinding.inflate(
+    ) = FragmentStandardUiBinding.inflate(
         inflater,
         container,
         false
     ).also {
+        viewModel.weatherInfo.observe(viewLifecycleOwner, { info ->
+            val recyclerView = (view as View).findViewById<RecyclerView>(R.id.mainRecyclerView)
```
Comment (ConstAlpha) [inline ui/standard-ui/src/main/java/com/ait/ui/standard/view/StandardUiFragment.kt:31]:
```suggestion
            val recyclerView = requireView().findViewById<RecyclerView>(R.id.mainRecyclerView)
```

File: ui/standard-ui/src/main/java/com/ait/ui/standard/view/StandardUiFragment.kt [hunk 9]
```diff
@@ -17,11 +22,17 @@ class StandardUiFragment : Fragment() {
...
+            recyclerView.layoutManager = LinearLayoutManager(context)
```
Comment (ConstAlpha) [inline ui/standard-ui/src/main/java/com/ait/ui/standard/view/StandardUiFragment.kt:32]: Can be moved into xml file

**Review by ConstAlpha — APPROVED** [review]
